tests/integration/star_wars/test_previous_link.py

```diff
@@ -1,89 +0,0 @@
-from tests.integration.star_wars.star_wars_tests import StarWarsTestCase
-from tests.integration.star_wars import star_wars_test_urls
-
-class TestPreviousLink(StarWarsTestCase):
-
-    def test_previous_link_path(self):
```

Comment:
The previous page is now just a href so no value testing that with integration tests but its covered in selenium